package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vets-json-schema",
-  "version": "6.7.2",
+  "version": "7.0.1",
```

Comment:
does this PR break backwards compatibility? if it's only adding features then a minor version bump should be ok

Reply:
Version is based on the PRs that are in the queue prior to this one @mchelen-gov if this PR is merging prior to those the Version would need to be update as this is a minor or even a patch increment.